examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -905,6 +943,9 @@ def main(args):
             " Please use `huggingface-cli login` to authenticate with the Hub."
         )
 
+    if args.do_edm_style_training and args.snr_gamma is not None:
+        raise ValueError("Min-SNR formulation is not supported when conducting EDM-style training.")
```

Comment:
do this earlier, so it doesn't load the model yet.

Reply:
It's at the beginning:

https://github.com/huggingface/diffusers/blob/59433ca1ae76445908e63fe4e2f9aceaf95baf0c/examples/dreambooth/train_dreambooth_lora_sdxl.py#L948

way before the model loading code.